internal/cli/channel_command.go

```
@@ -0,0 +1,332 @@
+	if err != nil {
+		return false, err
+	}
+	if channel == ChannelStable {
+		return false, nil
+	}
+	target, err := ChannelBinaryPath(channel)
+	if err != nil {
+		return false, err
+	}
+	if _, err := os.Stat(target); err != nil {
+		return false, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 97) but it returns nil

(nilerr)